### PR TITLE
Refactor library and sample to AndroidX packaging system.

### PR DESCRIPTION
**Pull request by pihariev**

### CODE REVIEW

File: sample/src/main/java/ru/terrakok/cicerone/sample/utils/moxy/MvpAppCompatActivity.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
+package ru.terrakok.cicerone.sample.utils.moxy;
+
+import android.annotation.SuppressLint;
+import android.os.Bundle;
+import androidx.appcompat.app.AppCompatActivity;
+import com.arellomobile.mvp.MvpDelegate;
+
+@SuppressLint({ "Registered", "unused" })
+public class MvpAppCompatActivity extends AppCompatActivity {
```
Comment (terrakok) [inline sample/src/main/java/ru/terrakok/cicerone/sample/utils/moxy/MvpAppCompatActivity.java]: Why do you copy-pasted this class? Jetifier don't work?

**Comment by terrakok** [comment]
Set target branch to `develop`

File: sample/src/main/java/ru/terrakok/cicerone/sample/utils/moxy/MvpAppCompatActivity.java (hunk 1 above)
Comment (pihariev) [reply sample/src/main/java/ru/terrakok/cicerone/sample/utils/moxy/MvpAppCompatActivity.java]: My bad, fixed it.

**Comment by pihariev** [comment]
> Set target branch to `develop`

Did a rebase to develop, do I need to make another PR?

**Comment by TheLester** [comment]
When this will be released?

**Comment by locskot** [comment]
Waiting for release! When?

**Comment by 8t89** [comment]
Any news on the release date?

**Comment by Xset-s** [comment]
Any news on the release date?

**Comment by Xset-s** [comment]
Anyyyy neews??

**Comment by TheLester** [comment]
@terrakok Hey, please make a release with at least this concrete feature

**Pull request merged**